WcaOnRails/app/views/competitions/payment_setup.html.erb

```diff
@@ -1,46 +1,86 @@
-      your own Stripe account with the WCA Stripe account by clicking the
-      button below. If you don't already have your own Stripe account you can
-      still click on the button below and it will guide you through the
-      process.
-    </p>
-    <p>
-      Connecting your Stripe account gives the WCA website the ability to
-      create charges, and issue refunds directly on your Stripe account.
-    </p>
+    <p><b><%= t('payments.payment_setup.stripe_connect_warning') %></b></p>
+    <p><%= t('payments.payment_setup.stripe_payment_info') %></p>
     <%= link_to @authorize_url, class: "btn btn-primary" do %>
-      <%= ui_icon("cc stripe") %> Connect with Stripe
+      <%= ui_icon("cc stripe") %><%= t('payments.payment_setup.connect_button', provider: t('payments.payment_providers.stripe')) %>
     <% end %>
 
+  <%# Stripe connected %>
   <% else %>
+    <p><b><%= t('payments.payment_setup.stripe_disconnect_warning') %></b></p>
     <p>
-      This competition is set up to accept payments through Stripe.
-      If you want to disconnect your Stripe account, you should first do so from your Stripe dashboard
-      <%= link_to "here", "https://dashboard.stripe.com/account/applications" %> (please note it will revoke access for all competitions using the account).
-      Before a new Stripe account can be connected, please request your Delegate to reach out to WCAT to clear the entry.
-    </p>
-    <p>
-      Keep in mind that these account changes should only be done in exceptional situations, such as the current account being unable to receive payments.
+      <%= t('payments.payment_setup.stripe_disconnect_info') %>
+      <ul><li><%= link_to(t('payments.payment_setup.stripe_dashboard'), "https://dashboard.stripe.com/account/applications") %></li></ul>
     </p>
 
+    <% if current_user&.can_admin_competitions? %>
+      <%= button_to(
+        competition_disconnect_stripe_path,
+        class: 'btn btn-danger',
+        disabled: @competition.connected_stripe_account_id.nil?,
+        method: :post, data: { confirm: t('payments.payment_setup.confirm_account_disconnect', provider: t('payments.payment_providers.stripe')) }
+      ) do %>
+        <%= t('payments.payment_setup.disconnect_account', provider: t('payments.payment_providers.stripe')) %>
+      <% end %>
+    <% end %>
+
   <% end %>
 
-  <% if current_user&.can_admin_competitions? %>
-    <%= button_to competition_disconnect_stripe_path, class: 'btn btn-danger', disabled: @competition.connected_stripe_account_id.nil?, method: :post, data: { confirm: t('competitions.stripe_disconnect_confirm') } do %>
-      <%= t 'competitions.stripe_disconnect' %>
+  <% unless Rails.env.production? %>
+    <%# PayPal Payments %>
+    <h2><%= t('payments.payment_setup.provider_heading', provider: t('payments.payment_providers.paypal')) %></h2>
+
+    <%# PayPal not connected %>
+    <% if !CompetitionPaymentIntegration.paypal_connected?(@competition) %>
```

Comment:
`if !foo` should always be `unless foo` for stylistic reasons.

Reply:
Also: This `if` has an else branch. So instead of
```ruby
if !foo # alias "unless foo", which is just syntactic sugar
  do_that()
else
  do_this()
end
```

you can and should do

```ruby
if foo
  do_this()
else
  do_that()
end
```